src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -297,7 +330,7 @@ private static boolean processArgs(String[] args, Set<ArgumentFlags> flags, Set<
 
     // Command-line argument flags
     private enum ArgumentFlags {
-        NIST, HYPO, LDC, PROGRAM, FILES, DIRECTORY, FILE_OUTPUT, PROFILING, PROGRESSIVE_PROFILING
+        NIST, HYPO, LDC, PROGRAM, FILES, DIRECTORY, FILE_OUTPUT, PROFILING, PROGRESSIVE_PROFILING, ABORT
```

Comment:
This is getting pretty big. It might be time to bite the proverbial bullet and make an Arguments class to house all the argument variables. In fact (it's not something I'd thought of in our discussion), but it could house the flags as well as other values like abortParam.

Reply:
Based on discussions, we'll defer this and other refactoring until after we have at least preliminary results from the progress monitor and have made a little headway on TA2 processing.  The approach will be to do the full refactor-- to break out the validator command line tool from the validator class that does the actual validation.